tests/integration/api_v3/http/consistency_stakers.test.js

```diff
@@ -25,7 +25,7 @@ for (let i = 1; i < baseUrls.length; i++) {
 	const baseUrl0 = baseUrls[0];
 	const baseUrl1 = baseUrls[i];
 
-	xdescribe('Votes Received', () => {
+	describe('Votes Received', () => {
```

Comment:
It should be stakers
```suggestion
	describe('Stakers', () => {
```